core/compiler.cpp

```diff
@@ -27,13 +27,35 @@
 namespace trtorch {
 namespace core {
 
+static std::unordered_map<int, runtime::CudaDevice> cuda_device_list;
```

Comment:
We need to move this into the runtime module, if people are using libtrtorchrt they will not have access to this code

Reply:
Moved this inside runtime.